submissions/xT0PERx/js-dom/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" type="text/css" href="visuality.css" />
+    <title>My interactive side-menu</title>
+  </head>
+  <body>
+    <div class="container clearfix">
+      <div class="menu">
+        <ul class="menu-list">
+          <li class="menu-title">О компании:</li>
```

Comment:
UL group equivalent elements, title is not equivalent to list element.

Even though they look the same on your website